backend/app/api/api_v1/routers/admin.py

```diff
@@ -90,7 +93,7 @@ async def user_edit(
     """Updates existing user"""
     updated_user = edit_user(db, user_id, user)
 
-    send_email(EmailType.account_changed, updated_user)
+    # send_email(EmailType.account_changed, updated_user)
```

Comment:
What's the reason for not sending an email when a user's account details are edited? Is this just a thing we'll activate later?

Reply:
yeah, basically we don't have a template yet :)